### PR TITLE
[14] Train classifiers

### LINKED ISSUE #14: Train 'Mission Classifiers'
Train 3 binary 'Mission Classifiers' i.e one classifier for AHL, AFS, ASF:
- Start with simple keyword matching approach
- Try more advanced methods such as embeddings + classifiers or fine-tuning a transformer model

If more training data is needed, we can use LLMs to label more data.

### PR DESCRIPTION
**Pull request by Jack-0-0**
Closes #14.

This PR has two notebooks that were used to explore training models to classify if text is relevant to Nesta's missions:
* `train_classifiers.ipynb` has models such as SVMs and logistic regression trained on embeddings of the text.
* `train_bert_classifier_colab.ipynb` trains a bert model. This file is to be used on colab using a GPU.

### CODE REVIEW

**Review by beingkk — APPROVED** [review]
Hey, thank you for this, I've skimmed through the notebooks and it looks good to merge!
